src/mds/MDLog.cc

```diff
@@ -581,6 +581,27 @@ class C_OFT_Committed : public MDSInternalContext {
   }
 };
 
+void MDLog::try_to_commit_open_file_table()
+{
+  ceph_assert(ceph_mutex_is_locked_by_me(submit_mutex));
+
+  if (capped) // shutting down the MDS
```

Comment:
nit: unrelated to this change - if `capped` is supposed to identify MDS shutting down, it's way more constructive to name it `shutdown`.

Reply:
Sure, will fix it.